docker-compose.yml

```diff
@@ -39,9 +39,9 @@ services:
     entrypoint:
       - /stolon-pgbouncer/bin/stolon-pgbouncer.linux_amd64
       - supervise
+      - --metrics-address=0.0.0.0
       - --enable-child-process
       - --pgbouncer-config-template-file=/stolon-pgbouncer/docker/stolon-development/pgbouncer/pgbouncer.ini.template
-      - --poll-interval=5s
```

Comment:
?

Reply:
Ah this was removed as it doesn't help our tests and makes the dashboard a bit strange as it assumes a minute value.

To be clearer, stolon-pgbouncer uses an etcd watch so should update immediately, which is why reducing this value wouldn't improve our CI times.